docs/resources/service_compute.md

```diff
@@ -105,6 +105,7 @@ $ terraform import fastly_service_compute.demo xxxxxxxxxxxxxxxxxxxx@2
 - **logging_splunk** (Block Set) (see [below for nested schema](#nestedblock--logging_splunk))
 - **logging_sumologic** (Block Set) (see [below for nested schema](#nestedblock--logging_sumologic))
 - **logging_syslog** (Block Set) (see [below for nested schema](#nestedblock--logging_syslog))
+- **reuse** (Boolean) Services that are active cannot be destroyed. This will cause the service to be deactivated allowing it to be reused by importing it into another Terraform project. Default `false`
```

Comment:
The first time I read this I thought it meant "Services that are active cannot be destroyed *if you set this property to true*".  However now I see the impl, I realise it doesn't mean that.  

```suggestion
- **reuse** (Boolean) Services that are active cannot be destroyed. If set to `true` a service Terraform intends to destroy will instead be deactivated (allowing it to be reused by importing it into another Terraform project). If `false`, attempting to destroy an active service will cause an error. Default `false`
```